docs/src/main/paradox/testing-actors.md

```diff
@@ -0,0 +1,44 @@
+Testing Actors
+--------------
+
+We will use the [ScalaTest](http://www.scalatest.org/) framework to conduct the testing of our sample. The test coverage is not complete in this example. We leave that as an exercise for the interested reader. Instead, the test code is there to showcase how how easy it is to test actor code and also to provide some basic concepts.
+
+### Test Class Definition
+
+Let's start by looking at the test class definition:
+
+@@snip [HelloAkka.scala]($g8srctest$/scala/com/lightbend/akka/sample/HelloAkkaSpec.scala) { #test-classes }
+
+The test class extends `akka.test.TestKit` which is a module for unit _and_ integration testing of actors and actor systems. The showcased test class only uses a fraction of the functionality provided by [TestKit](http://doc.akka.io/docs/akka/2.5/scala/testing.html). The rest of the classes extended are from ScalaTest and the provide syntax and functionality that we use in our test cases. We will use `FlatSpecLike` with with we can write test specifications in the format of "X should Y", e.g.:
+
+@@snip [HelloAkka.scala]($g8srctest$/scala/com/lightbend/akka/sample/HelloAkkaSpec.scala) { #specification-example-1 }
+
+and
+
+@@snip [HelloAkka.scala]($g8srctest$/scala/com/lightbend/akka/sample/HelloAkkaSpec.scala) { #specification-example-2 }
+
+### Test Methods
+
+Now that we know some of the basic concepts it is time to look at the implemented test methods. As described above `TestKit` enables us to do both unit and integration testing. Our test class uses both approaches.
+
+#### Unit Testing
```

Comment:
Will remove the section about unit testing and only focus on the integration testing part.